### ISSUE TITLE
0.54.1 release has wrong date

### ISSUE THREAD

**Issue by onwsk8r** [prometheus-operator/prometheus-operator#4615]
**What happened?**

2022 happened, can you believe it!?

**Did you expect to see something different?**

I don't judge, but it certainly threw my script for a loop

**How to reproduce it (as minimally and precisely as possible)**:

1. Remain in a single year for at least 365 days. Ensure EUs type the year at least monthly.
2. Allow the year to increment naturally (this change **must** be monotonic: the year **must not** have been used before)
3. Monitor dates typed by EUs, perhaps with a solution like Prometheus
4. Observe high error rates that will self-correct over time

**Environment**

* Prometheus Operator version:

```shell
❯ curl -fsSL https://api.github.com/repos/prometheus-operator/prometheus-operator/releases/latest | jq -r '.name'
0.54.1 / 2021-02-24
```

* Kubernetes version information:

```
❯ kubectl version | sed -ne 's/\([[:alpha:]]\+\):/"\1":/g' -e '/^Server/ s/^[^{]\+//p' | jq
{
  "Major": "1",
  "Minor": "21",
  "GitVersion": "v1.21.6",
  "GitCommit": "d921bc6d1810da51177fbd0ed61dc811c5228097",
  "GitTreeState": "clean",
  "BuildDate": "2021-10-27T17:44:26Z",
  "GoVersion": "go1.16.9",
  "Compiler": "gc",
  "Platform": "linux/arm64"
}
```

* Kubernetes cluster kind:

Kubespray

**Anything else we need to know?**:

**Comment by fpetkovski** [comment]
Time goes backwards in our universe :smiling_face_with_tear:. Looks like we have the same problem with 0.54.0. Will submit a patch to fix the changelog and tweak the releases.

**Comment by PhilipGough** [comment]
Closed by #4616
